src/lib/hash.h

```diff
@@ -85,4 +86,16 @@ pgp_hash_alg_t pgp_hash_alg_type(const pgp_hash_t *hash);
 
 pgp_hash_alg_t pgp_str_to_hash_alg(const char *);
 
+/* -----------------------------------------------------------------------------
+ * @brief   Returns output size of an digest algorithm
+ *
+ * @param   [in] alg
+ * @param   [out] output_length: must be not NULL
+ *
+ * @return  true if provided algorithm is supported and it's size was
+ *          correctly retrieved, otherwise false
+ *
+-------------------------------------------------------------------------------- */
+bool pgp_hash_digest_length(pgp_hash_alg_t alg, size_t *output_length);
```

Comment:
Yup, right function name is stupid - it must be called pgp_digest_length. And it should return false if invocation fails as stated in descripiton